src/config/dynamic-theme-fixes.config

```diff
@@ -1791,7 +1791,10 @@ code:first-of-type {
 .sc-186t4q4-1 {
     background-color: ${rgba(3, 9, 15, 0.02)} !important;
 }
-#ak-main-content {
+div.zgsxji-0.jMIyWM > div > div > div > div {
+    background-color: var(--darkreader-bg--ds-surface, #181a1b) !important;
+}
+#ak-main-content, html, ._11q7vuon {
```

Comment:
Why is `html` being added here? That's quite a risky fix for such a big site.

Reply:
@Gusted, I couldn't figure out any other way to get the background of the bottom of the page to change color:

Without `html`:
<img width="1527" alt="image" src="https://user-images.githubusercontent.com/1766174/190859995-07e811c4-7e27-4d7a-8708-4f27498c43bf.png">

With `html`:
<img width="1524" alt="image" src="https://user-images.githubusercontent.com/1766174/190860010-c1f84ad3-5631-4475-a924-d774206cea8e.png">


I'm certainly open to suggestions for how to make this less risky, I'm just not necessarily sure how to!

Reply:
```suggestion
#ak-main-content, .pfb8c3-0.dpzKHj, ._11q7vuon {
```
This is less ideal, but also less risky: 
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/1766174/190886093-630d6a1f-9a86-4638-a365-fc1331db8056.png">